app/master/build_request_handler.py

```diff
@@ -148,7 +149,13 @@ def _compute_subjobs_for_build(self, build_id, job_config, project_type):
         :type project_type: project_type.project_type.ProjectType
         :rtype: list[Subjob]
         """
-        atoms_list = job_config.atomizer.atomize_in_project(project_type)
+        # Users can override the list of atoms to be run in this build. If the atoms_override
+        # was specified, we can skip the atomization step and use those overridden atoms instead.
+        if project_type.atoms_override is not None:
+            atoms_string_list = project_type.atoms_override
+            atoms_list = [Atom(atom_string_value) for atom_string_value in atoms_string_list]
```

Comment:
It's funny you should ask.

For the feature that I want to build, I'd like it if we:
- Ran the atomization first.
- Checked to see if the atoms_override list is a proper subset of the original atomization.
- Only run the atoms_override that is a subset of both the atoms_override and original atomization.

However, I feel that this behavior is unexpected given the API, and that I do believe the existing functionality (slave fails if the atom is invalid, and essentially treat it as we would a bad atomization command section).